src/repos/ArticleRepo.ts

```diff
@@ -103,6 +116,12 @@ const incrementShoutouts = async (id: string): Promise<Article> => {
   return article.save();
 };
 
+const isCovidArticle = (title: string): boolean => {
+  const filter = new Filter();
+  filter.addWords("covid", "covid-19", "coronavirus", "test", "mask", "masks", "pandemic", "tests", "omicron","endemic")
```

Comment:
While the goal currently is to remove COVID articles, I don't think the code should have a notion that it's removing COVID articles - hopefully COVID will be gone soon and volume-backend will be around for longer. Generally, the task here is to remove "bad" articles and currently "bad" for us are articles with COVID related content. I suggest 2 changes: 1. Move all the "bad" title words to a list of words in the constants file. 2. Rename the method `isCovidArticle` to `isBadArticle` or something similar. This way when we no longer care to remove COVID content, rather than deprecate the function, you just remove the words in the constants file and you don't have to remove the function/task of removing "bad" articles. If one day Apple, comes along and says we have to remove all articles related to Microsoft - you don't change the function, you just add "Microsoft" to the list of constant words.

Reply:
@chalo2000 @dishanest feel free to comment on this